src/libs/ComposerUtils/debouncedSaveReportComment.js

```diff
@@ -0,0 +1,13 @@
+import _ from 'underscore';
+import * as Report from '../actions/Report';
+
+/**
+ * Save draft report comment. Debounced to happen at most once per second.
+ * @param {String} reportID
+ * @param {String} comment
+ */
+const debouncedSaveReportComment = _.debounce((reportID, comment) => {
```

Comment:
This caused a regression of showing draft icon - more information here - https://github.com/Expensify/App/issues/27362#issuecomment-1717803795